testdata/ansible/memcached-operator/config/default/kustomization.yaml

```diff
@@ -18,6 +18,11 @@ resources:
 - ../crd
 - ../rbac
 - ../manager
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
```

Comment:
Are we starting to support webhooks and cert-manager by default in ansible and helm operators? If so, this again needs a migration guide entry.

Reply:
Let me check on this.

Reply:
same like this - https://github.com/operator-framework/operator-sdk/pull/6047#discussion_r993834871

Reply:
Updated